.github/workflows/windows-builds-on-pr.yaml

```diff
@@ -26,6 +26,8 @@ jobs:
           - target: i686-pc-windows-gnu
             mingwdir: mingw32
             mingw: https://ci-mirrors.rust-lang.org/rustc/i686-6.3.0-release-posix-dwarf-rt_v5-rev2.7z
+          - target: aarch64-pc-windows-msvc
```

Comment:
Oh, oops, it actually did build after all 😅 Nice verification though! Should we then keep it enabled at least for Stable?